src/compiler/crystal/macros/macros.cr

```diff
@@ -145,7 +145,7 @@ class Crystal::Program
 
       # Although release takes longer, once the bc is cached in .crystal
       # the subsequent times will make program execution faster.
-      host_compiler.release = true
+      host_compiler.release!
```

Comment:
**note:** According to https://github.com/crystal-lang/crystal/issues/13505#issuecomment-1567068656 ff. it seems to be more efficient to build macros not in a single module.
It's probably best to leave the current behaviour in place here. We can follow up with a change to macro generation config. Since the host compiler inherits its configuration from the target compiler there's more involved than just switching this to `host_compiler.optimization_mode = :03`.

Reply:
Not sure about changing this, because this is for compile `run` macroses, as i understand. Which should be executed fast, and this is done only once.

Reply:
@kostya macroses should run "fast-enough". Compiled without "single-module" is certainly fast enough for macroses, since they are not computation heavy. I doubt you could measure difference, I bet box of bear on it. But delta of time consumed by compilation of macroses is certainly measurable.